Build error on Arch Linux

Building from the latest AUR 3.0.6 package crashes with the following error:
```
[ 44%] Building CXX object src/CMakeFiles/DownZemAll.dir/core/mimedatabase.cpp.o
In file included from /home/papavlos/.cache/yay/downzemall/src/DownZemAll-3.0.6/src/core/mimedatabase.cpp:17:
/home/papavlos/.cache/yay/downzemall/src/DownZemAll-3.0.6/src/core/mimedatabase.h:35:35: error: ‘QUrl’ does not name a type
   35 |     static QPixmap fileIcon(const QUrl &url, int extend = default_icon_size);
      |                                   ^~~~
/home/papavlos/.cache/yay/downzemall/src/DownZemAll-3.0.6/src/core/mimedatabase.cpp:85:9: error: no declaration matches ‘QPixmap MimeDatabase::fileIcon(const QUrl&, int)’
   85 | QPixmap MimeDatabase::fileIcon(const QUrl &url, int extend)
      |         ^~~~~~~~~~~~
```
I run Plasma desktop environment and have already built some other Qt-based packages without issues. Is there a missing dependency?
**yochananmarqos** - the maintainer of the AUR packages - also confirms the same error and lets me open this upstream issue here.
See: [https://aur.archlinux.org/packages/downzemall](https://aur.archlinux.org/packages/downzemalll)

--- a/src/core/mimedatabase.h
+++ b/src/core/mimedatabase.h
@@ -20,6 +20,7 @@
 
 #include <QtCore/QString>
 #include <QtCore/QStringList>
+#include <QtCore/QUrl>
 #include <QtGui/QPixmap>
 
 constexpr int default_icon_size = 32;

